### PR TITLE
[Translation] More detailed yaml parse exception message

### PR DESCRIPTION
**Pull request by tgabi333**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets |  |
| License | MIT |
| Doc PR |  |

### CODE REVIEW

File: src/Symfony/Component/Translation/Loader/YamlFileLoader.php [hunk 1]
```diff
@@ -54,7 +54,7 @@ public function load($resource, $locale, $domain = 'messages')
         try {
             $messages = $this->yamlParser->parse(file_get_contents($resource));
         } catch (ParseException $e) {
-            throw new InvalidResourceException(sprintf('Error parsing YAML, invalid file "%s"', $resource), 0, $e);
+            throw new InvalidResourceException(sprintf('Error parsing YAML, invalid file "%s" at line: %d', $resource, $e->getParsedLine()), 0, $e);
```
Comment (fabpot) [inline src/Symfony/Component/Translation/Loader/YamlFileLoader.php]: Can you remove the colon in the message (`at line %d`)?

**Comment by fabpot** [comment]
:+1:

**Comment by aitboudad** [comment]
:+1:

File: src/Symfony/Component/Translation/Loader/YamlFileLoader.php [hunk 2]
```diff
@@ -54,7 +54,7 @@ public function load($resource, $locale, $domain = 'messages')
         try {
             $messages = $this->yamlParser->parse(file_get_contents($resource));
         } catch (ParseException $e) {
-            throw new InvalidResourceException(sprintf('Error parsing YAML, invalid file "%s"', $resource), 0, $e);
+            throw new InvalidResourceException(sprintf('Error parsing YAML, invalid file "%s" at line %d', $resource, $e->getParsedLine()), 0, $e);
```
Comment (aitboudad) [inline src/Symfony/Component/Translation/Loader/YamlFileLoader.php]: we should check line value `>= 0` see [ParseException.php#L136-L138](https://github.com/symfony/symfony/blob/2.7/src/Symfony/Component/Yaml/Exception/ParseException.php#L136-L138)  or even use ParseException :
```php
$parseException = new ParseException('Error parsing YAML', $e->getParsedLine(), $e->getSnippet(), $resource);
throw new InvalidResourceException($parseException->getMessage());
```

Comment (tgabi333) [inline src/Symfony/Component/Translation/Loader/YamlFileLoader.php]: i think we should not replicate logic from ParseException, just use it's message.

**Comment by aitboudad** [comment]
@tgabi333  well I think we just duplicate ParseException message here, if someone want more details just see the previous exception.

selection_002

**Comment by Tobion** [comment]
I agree with @aitboudad. :-1: to this PR.

**Pull request closed without merging**